### DISCUSSION TITLE
Is it available to bind two sequencial keys to one action?

### DISCUSSION THREAD

**Discussion by linrongbin16** [junegunn/fzf#3336] [Q&A] [answered]
I already know how to bind key to fzf action:

for example: `--bind ctrl-d:page-down`.

But now I want to bind two sequential keys to an action, for example:
```
--bind ctrl-w+ctrl-d:page-down
```
Is that available? first press ctrl-w, then ctrl-d to page down?

**Comment by junegunn** [comment, marked as answer]
No, there is no plan to support it either. If you need it for Vim plugins, consider using tmaps of Vim or Neovim instead.

**Reply by linrongbin16** [reply]
thanks
